### ISSUE TITLE
An idea for the "plugin" approach - use repo2docker config files

### ISSUE THREAD

**Issue by choldgraf** [yuvipanda/tljh-pangeo#1]
One thought - what if we tried to completely co-opt the "repo2docker" config file structure. E.g. repo2docker goes from repository -> Dockerfile -> image. This tool could look for the same repository files (except for a Dockerfile) and install the environment specified in it. That way we're not telling people to use a new pattern of interaction. You could do `tljh-config install path_to_repo` or something?

**Comment by yuvipanda** [comment]
I agree! I think it should be a plugin itself rather than be part of tljh cor.e
